drivers/hmis_external_apis/lib/tasks/setup.rake

```diff
@@ -0,0 +1,26 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# rails driver:hmis_external_apis:seed_ac_hmis_unit_types
+desc 'Seed AC HMIS Unit Types from JSON file'
+task seed_ac_hmis_unit_types: [:environment, 'log:info_to_stdout'] do
+  mper_creds = GrdaWarehouse::RemoteCredentials::Smtp.where(slug: 'mper').first!
```

Comment:
I don't think this should be `Smtp` It probably needs it's own sub-class.

Reply:
Ok, I wasn't sure. That's just how I have the stub set up locally. Can you add any comments on what that subclass should look like to https://www.pivotaltracker.com/story/show/185000178
thank you!

Reply:
I'm just going to leave this here until the ticket is implemented